CHANGELOG.md

```diff
@@ -85,19 +89,24 @@ for Rust libraries in [RFC #1105](https://github.com/rust-lang/rfcs/blob/master/
 * Support for `bigdecimal` < 0.0.13 has been removed.
 * Support for `pq-sys` < 0.4.0 has been removed.
 * Support for `mysqlclient-sys` < 0.2.5 has been removed.
-* Support for `time` types has been removed.
+* Support for `time` (0.1) types has been removed.
 * Support for `chrono` < 0.4.19 has been removed.
+* The minimal supported version of libsqlite3-sys is now 0.17.2.
 * The `NonNull` trait for sql types has been removed in favour of the new `SqlType` trait.
 * `no_arg_sql_function!` has been deprecated without replacement.
   [`sql_function!`][sql-function-2-0-0] can now be used for functions with zero
   arguments. See [the migration guide][2-0-migration] for more details.
 * Support for `barrel` based migrations has been removed for now. We are happy to
   add this support back as soon as `barrel` integrates with our new migration framework.
-* Deprecated bash completions command (`diesel bash-completions`) has been removed.
+* The deprecated bash completions command (`diesel bash-completions`) has been removed.
```

Comment:
I am personally guilty of writing this changelog entry but now I think it's redundant given the very first line, which states
"Everything previously deprecated has been removed".